packages/taucmdr/cli/commands/trial/tests/test_create.py

```diff
@@ -137,6 +137,16 @@ def test_heap_usage_memory_alloc_profile(self):
         self.assertInLastTrialData("compute")
         self.assertInLastTrialData("malloc")
 
+    def test_without_libdwarf(self):
```

Comment:
This test looks good. It would be nice to have tests that verify that libdwarf is not passed to TAU's configure script when `--libdwarf none` and that it is under normal circumstances (on Linux). I'm trying to think of the best place to put such a test, but it's not immediately obvious to me where it should go. I'll think about this some more and then once I have a better idea I can point you in the right direction or implement it myself. This shouldn't block this PR, so I'll put it on my todo list.

Reply:
I think to do this we'll need a new assert function that can examine the output of configuring a package and check for the presence or absence of a string. If libdwarf is used, for example, TAU's configure script will print `Using libdwarf at <path>` and `NOTE: Using DWARF support from libdwarf` while without libdwarf it won't. But there's not currently a convenient way to check the output of configure during a test, as far as I know.

Reply:
The configuring usually happens when `reset_project_storage()` but you could add a target and switch to it and capture that output more easily. One problem with this approach is that it's not guaranteed to require being reconfigured.